tasks/config.yml

```diff
@@ -238,11 +238,11 @@
   - restart freeswitch
   - reload systemd
 
-- name: configure freeswitch to use correct IP address for wss-binding
+- name: configure wss-binding of FreeSWITCH
```

Comment:
I would like a more verbose output, would suggest adding the `{{ bbb_freeswitch_ip_address }}` to the `- name:`.
This way without higher verbosity the user can see which IP got set.

```yaml
- name: configure IP {{ bbb_freeswitch_ip_address }} for wss-binding of FreeSWITCH
```